tools/deprecate_version/deprecate_version.py

```diff
@@ -20,24 +19,18 @@
 # Known issues:
 # - Minor fixup PRs (e.g. fixing a typo) will result in the creation of spurious
 #   issues.
-# - Later PRs can clobber earlier changed to DEPRECATED.md, meaning we miss
-#   issues.
 
 from __future__ import print_function
 
 from collections import defaultdict
 import os
 import re
+import subprocess
 import sys
 
 import github
 from git import Repo
 
-try:
```

Comment:
did we move completely to python3 on this?

Reply:
I thought we'd switched our tools to 3 by default, and I handled a bunch of string-bytestring compat issues I think were Python 3, and it seems to work without this try phrase.  I can add it back if that's not enough?

Reply:
From https://github.com/envoyproxy/envoy/blob/master/tools/deprecate_version/deprecate_version.sh I don't think it is forced to run under Python 3? Python 2 `input` is interpreting the input as python script, (i.e. `eval(raw_input())`), so if not sure let's keep this. Or force Python 3 in the shell script as this one is likely only run by maintainer.